BlotoutAnalytics/Foundation/BOFUtilities/BOReachability.m

```diff
@@ -116,25 +116,25 @@ @implementation BOReachability
 
 + (instancetype)reachabilityWithAddress:(const struct sockaddr_in *)hostAddress;
 {
-	SCNetworkReachabilityRef reachability = SCNetworkReachabilityCreateWithAddress(kCFAllocatorDefault, (const struct sockaddr *)hostAddress);
+    BOReachability* returnValue = [[self alloc] init];;
 
-	BOReachability* returnValue = NULL;
-
-	if (reachability != NULL)
-	{
-		returnValue = [[self alloc] init];
-		if (returnValue != NULL)
-		{
-			returnValue->reachabilityRef = reachability;
-			returnValue->localWiFiRef = NO;
-		}
-	}
-	return returnValue;
+    if (returnValue != NULL)
```

Comment:
this code is wrong, once object is created we didn't have network change information

Reply:
Since the calling function expecting reachability null in case no network , so need to pass null otherwise it got stuck or need to do some heavy changes . So recall this issue and changes.